code/web/src/modules/user/api/actions.js

```diff
@@ -160,3 +158,31 @@ export function getUser(id) {
     })
   }
 }
+
+// get user's products
+export function getUserProducts(token) {
+  return dispatch => {
```

Comment:
I will need your help in my request to the server for the updated form. I'll review this code and use it for mine.